resources/assets/components/Campaign/index.js

```diff
@@ -109,6 +110,14 @@ const Campaign = ({ id }) => {
           <p>–</p>
         )}
 
+        <h4>Campaign Group Type ID</h4>
+        {campaign.contentfulCampaignId ? (
+          <a href={`/group-types/${groupTypeId}`} target="_blank">
+            Group Type Id{campaign.groupTypeId}
```

Comment:
Let's just display the `{campaign.groupTypeId}` value here to keep consistent with some of the other internal ID links around the site.
```suggestion
           {campaign.groupTypeId}
```